fedimint-rocksdb/src/lib.rs

```diff
@@ -28,14 +31,46 @@ impl RocksDb {
     }
 }
 
+fn is_power_of_two(num: usize) -> bool {
+    num != 0 && (num & (num - 1)) == 0
+}
```

Comment:
I think this would be easier to reason about if it used [`count_ones`](https://doc.rust-lang.org/stable/std/primitive.usize.html#method.count_ones):

```rust
num.count_ones() == 1
```

Reply:
Agreed, it was a bit of a puzzle. The current impl seems to be a common bit twiddle hack, so if we don't modify to use `count_ones`, perhaps a comment with a reference would help our future selves.

https://stackoverflow.com/a/600306

Reply:
I was rushing to wrap it and just googled / asked chatgpt and slapped some tests on top. :D